### PR TITLE
fix(KFLUXINFRA-777): Changing NodeSelector and Toleration

### PR DESCRIPTION
**Pull request by manish-jangra**
We're replacing nodes with EBS Volumes to the nodes with Local NVMe Disks as default node pool.

### CODE REVIEW

**Review by bamachrn — APPROVED** [review]
LGTM

File: configs/nvme-storage-configurator/staging/patch.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+- op: replace 
+  path: "/spec/template/spec/nodeSelector" 
+  value:
+    konflux-ci.dev/workload: konflux-tenants
+
+
```
[the remark this answers is not in the document]
Comment (manish-jangra) [reply configs/nvme-storage-configurator/staging/patch.yaml]: Thanks to your microscopic assessment, I have removed one of thise line.

`yamllint configs/nvme-storage-configurator/staging/patch.yaml` throws no error now. There quite a few earlier.

**Comment by manish-jangra** [comment]
> @manish-jangra Can you please associate this change with a JIRA issue ?

@eisraeli attached the jira item now.

**Review by hugares — APPROVED** [review]
/gltm

**Comment by psturc** [comment]
```
Unable to connect to the server: net/http: TLS handshake timeout
```
/retest-required

**Pull request merged**